src/connectedmachine/azext_connectedmachine/generated/_params.py

```diff
@@ -94,9 +94,7 @@ def load_arguments(self, _):
         c.argument('type_', options_list=['--type'], type=str, help='Specifies the type of the extension; an example '
                    'is "CustomScriptExtension".')
         c.argument('type_handler_version', type=str, help='Specifies the version of the script handler.')
-        c.argument('enable_auto_upgrade', arg_type=get_three_state_flag(), help='Indicate whether the extension '
-                   'should be automatically upgraded by the platform if there is a newer version available.')
-        c.argument('auto_upgrade_minor', arg_type=get_three_state_flag(), help='Indicate whether the extension should '
+        c.argument('auto_upgrade_minor', arg_type=get_three_state_flag(), help='Indicates whether the extension should '
```

Comment:
Will breaking change occur after you removed this parameter `enable_auto_upgrade`?